include/inexor/vulkan-renderer/application.hpp

```diff
@@ -16,30 +16,25 @@
 // forward declarations
 namespace inexor::vulkan_renderer::input {
 class KeyboardMouseInputData;
-} // namespace inexor::vulkan_renderer::input
+}
 
 namespace inexor::vulkan_renderer {
 
 class Application : public VulkanRenderer {
-    std::string m_application_name;
-    std::string m_engine_name;
-
-    std::uint32_t m_application_version{0};
-    std::uint32_t m_engine_version{0};
-
     std::vector<std::string> m_vertex_shader_files;
     std::vector<std::string> m_fragment_shader_files;
     std::vector<std::string> m_texture_files;
+    std::vector<std::string> m_shader_files;
```

Comment:
I think a rebase has gone wrong?

Reply:
Yes, I did this manually. I will resolve it.

Reply:
Done.